src/manifest_test.ts

```diff
@@ -99,6 +99,100 @@ Deno.test("Manifest() automatically registers types used by function input and o
   });
 });
 
+Deno.test("Manifest() properly converts name to proper key", () => {
+  const UsingName = DefineType({
+    name: "Using Name",
+    type: Schema.types.boolean,
+  });
+
+  const Function = DefineFunction(
+    {
+      callback_id: "test_function",
+      title: "Function title",
+      source_file: "functions/test_function.ts",
+      input_parameters: {
+        properties: { aType: { type: UsingName } },
+        required: [],
+      },
+    },
+  );
```

Comment:
We don't need this function. Instead we can add `types: [UsingName]` to the manifest to keep the scope of the test limited